sdk/ai/Azure.AI.Projects/src/Custom/Agent/AzureFunctionToolDefinition.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
```

Comment:
nit: Please remove this, as it is not an auto-generated file.

Reply:
Fixed.